### DISCUSSION TITLE
Steam Big Picture

### DISCUSSION THREAD

**Discussion by superk0ch** [LizardByte/community#342] [Sunshine] [answered]
Hello World.
After updating Sunshine (Windows) to the new Version Steam is only starting in Desktop Mode. I use Moonlight on Android as Client. Is anyone there who can give me a Clue?

**Comment by superk0ch** [comment]
Sorry. Found out thats a Problem with Moonlight on my Device.

**Comment by ReenigneArcher** [comment, marked as answer]
It's a bug in steam actually.

https://steamcommunity.com/groups/SteamClientBeta/discussions/3/3948029352678585138/?tscn=1700257020

**Reply by superk0ch** [reply]
Thank you very much. As it turned out, it only worked once, and then again it didn't.
